types/react-native/index.d.ts

```diff
@@ -9231,7 +9221,8 @@ export var EdgeInsetsPropType: React.Requireable<any>
 export var PointPropType: React.Requireable<any>
 
 declare global {
-    const global: GlobalStatic;
+    var global: GlobalStatic;
+
```

Comment:
This fixes `error TS2451: Cannot redeclare block-scoped variable 'global'.` when compiling projects without the "DOM" library in `tsconfig.json`. I'm not sure we even need to do this, but I don't know the history. I would think that we can just populate the `declare global` block with everything in `GlobalStatic` and remove the interface?